<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>520</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>420</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">KzScrcpy</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>

   <!-- 主内容区 -->
   <item>
    <widget class="QWidget" name="contentWidget" native="true">
     <layout class="QVBoxLayout" name="contentLayout">
      <property name="spacing">
       <number>24</number>
      </property>
      <property name="leftMargin">
       <number>28</number>
      </property>
      <property name="topMargin">
       <number>28</number>
      </property>
      <property name="rightMargin">
       <number>28</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>

      <!-- 标题区 -->
      <item>
       <layout class="QHBoxLayout" name="headerLayout">
        <property name="spacing">
         <number>12</number>
        </property>
        <item>
         <spacer name="headerLeftSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size><width>40</width><height>20</height></size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="logoLabel">
          <property name="text">
           <string>KzScrcpy</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="headerRightSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size><width>40</width><height>20</height></size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>

      <!-- 快速连接按钮 -->
      <item>
       <widget class="QFrame" name="connectCard">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <layout class="QVBoxLayout" name="connectCardLayout">
         <property name="spacing">
          <number>16</number>
         </property>
         <property name="leftMargin">
          <number>20</number>
         </property>
         <property name="topMargin">
          <number>20</number>
         </property>
         <property name="rightMargin">
          <number>20</number>
         </property>
         <property name="bottomMargin">
          <number>20</number>
         </property>
         <item>
          <widget class="QLabel" name="connectTitle">
           <property name="text">
            <string>快速连接</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="btnLayout">
           <property name="spacing">
            <number>16</number>
           </property>
           <item>
            <widget class="QPushButton" name="usbConnectBtn">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>1</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumHeight">
              <number>52</number>
             </property>
             <property name="text">
              <string>USB 连接</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="wifiConnectBtn">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>1</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumHeight">
              <number>52</number>
             </property>
             <property name="text">
              <string>WiFi 连接</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>

      <!-- 设备列表 -->
      <item>
       <widget class="QFrame" name="deviceCard">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <layout class="QVBoxLayout" name="deviceCardLayout">
         <property name="spacing">
          <number>14</number>
         </property>
         <property name="leftMargin">
          <number>20</number>
         </property>
         <property name="topMargin">
          <number>20</number>
         </property>
         <property name="rightMargin">
          <number>20</number>
         </property>
         <property name="bottomMargin">
          <number>20</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="deviceHeader">
           <property name="spacing">
            <number>12</number>
           </property>
           <item>
            <widget class="QLabel" name="deviceTitle">
             <property name="text">
              <string>设备列表</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="deviceHint">
             <property name="text">
              <string>双击连接</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="deviceSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size><width>40</width><height>20</height></size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QCheckBox" name="autoUpdatecheckBox">
             <property name="text">
              <string>自动刷新</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="updateDevice">
             <property name="minimumSize">
              <size><width>72</width><height>34</height></size>
             </property>
             <property name="text">
              <string>刷新</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QListWidget" name="connectedPhoneList">
           <property name="minimumHeight">
            <number>100</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>

   <!-- 底部工具栏 -->
   <item>
    <widget class="QFrame" name="toolbarFrame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="minimumHeight">
      <number>64</number>
     </property>
     <property name="maximumHeight">
      <number>64</number>
     </property>
     <layout class="QHBoxLayout" name="toolbarLayout">
      <property name="spacing">
       <number>16</number>
      </property>
      <property name="leftMargin">
       <number>28</number>
      </property>
      <property name="topMargin">
       <number>12</number>
      </property>
      <property name="rightMargin">
       <number>28</number>
      </property>
      <property name="bottomMargin">
       <number>16</number>
      </property>
      <item>
       <spacer name="toolbarLeftSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size><width>40</width><height>20</height></size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="settingsBtn">
        <property name="minimumSize">
         <size><width>110</width><height>40</height></size>
        </property>
        <property name="text">
         <string>⚙  设置</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="terminalBtn">
        <property name="minimumSize">
         <size><width>110</width><height>40</height></size>
        </property>
        <property name="text">
         <string>⌨  终端</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="toolbarRightSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size><width>40</width><height>20</height></size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
